SRC/NBASIC/nbas111e.txt

```diff
@@ -4,63 +4,63 @@ By Nestor Soriano (Konami Man), December 2.004
 
 1. WHAT IS NESTORBASIC?
 
-NestorBASIC is a set of machine code routines, integrated in a single file. 
+NestorBASIC is a set of machine code routines, integrated in a single file.     
 It is intended for being used within MSX-BASIC programs. Without losing 
 Turbo-BASIC compatibility, NestorBASIC gives you the following capabilities:
 
-- Full access to all te available memory of the computer (all existing memory 
-  in the case of DOS 1, all free memory in the case of DOS 2), up to 4 Mb.
-- Access to VRAM, with data block exchange between VRAM and between RAM and 
+- Full access to all the available memory of the computer (all existing memory  
+in case of DOS 1, all free memory in the case of DOS 2), up to 4 Mb.
+- Access to all VRAM, with data block exchange within VRAM and between RAM and 
   VRAM feature.
-- Storage of BASIC programs in the mapped RAM, which can be executed without 
-  losing the existing variables.
-- Access to disk files and direct access to physical sectors, with read/write 
-  to RAM/VRAM feature. File search, directory management. 
-- Graphic compression and decompression.
-- Moonblaster music replay. Samplekit load.
-- PSG sound effects replay.
-- Execution of machine code routines placed on BIOS, SUB-BIOS, normal BASIC 
-  memory, system work area, or a mapped RAM segment.
-- Machine code routines execution, from BIOS or from any RAM segment. User 
+- Allocating BASIC programs in the mapped RAM, with the capability to jump
+  from one program to another without losing variables.
+- Access to disk files and direct access to physical sectors, with
+  direct read/write to/from mapped RAM and VRAM feature. File search,
+  directory management. 
+- Graphics compression and decompression.
+- Moonblaster music playback. Samplekit loading.
+- PSG sound effects playback.
+- Executing machine code routines located in BIOS, SUB-BIOS, main BASIC 
+  memory, system working area, or a mapped RAM segment. User 
   defined interrupts.
 - NestorMan functions, InterNestor Suite and InterNestor Lite routines 
   execution.
 
-All of these functions are available through a single USR and an integer 
-parameter array, therefore they are fully TurboBASIC compatibles. In fact, 
-the TurboBASIC compiler itself is included into the NestorBASIC file, and it 
-is automatically loaded when NestorBASIC is installed.
+All of these functions are available through a single USR and an integer        
+parameter array, therefore they are fully TurboBASIC compatible. In fact,
+the TurboBASIC compiler itself is included into the NestorBASIC file, and it
+is automatically loaded when NestorBASIC is installed resident.

```

Comment:
I don't think the word "resident" is needed here. It's pretty clear from the rest of the documentation that NestorBASIC is a resident program.